tests/components/wunderground/test_sensor.py

```diff
@@ -91,9 +97,8 @@
     assert state.state == "Clear"
     assert state.name == "Weather Summary"
     assert "unit_of_measurement" not in state.attributes
-    assert (
-        state.attributes["entity_picture"] == "https://icons.wxug.com/i/c/k/clear.gif"
-    )
+    assert (state.attributes["entity_picture"] ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4751290)